src/test/java/org/elasticsearch/cluster/BlockClusterStatsTests.java

```diff
@@ -41,7 +44,7 @@
 
```

Comment:
I have the feeling that this test became obsolete after all the specific tests that you added. Maybe we can remove this class entirely?